packages/plugin/webpack/src/WebpackConfig.ts

```diff
@@ -163,114 +180,122 @@ export default class WebpackConfigGenerator {
     );
   }
 
-  async getPreloadConfigForEntryPoint(entryPoint: WebpackPluginEntryPointLocalWindow | WebpackPluginEntryPointPreloadOnly): Promise<Configuration> {
-    if (!entryPoint.preload) {
-      return {};
+  async getRendererConfig(entryPoints: WebpackPluginEntryPoint[]): Promise<Configuration[]> {
+    const entryPointsForTarget = {
+      web: [] as (WebpackPluginEntryPointLocalWindow | WebpackPluginEntryPoint)[],
+      electronRenderer: [] as (WebpackPluginEntryPointLocalWindow | WebpackPluginEntryPoint)[],
+      electronPreload: [] as (WebpackPluginEntryPointPreloadOnly | WebpackPluginEntryPointLocalWindow)[],
+      sandboxedPreload: [] as (WebpackPluginEntryPointPreloadOnly | WebpackPluginEntryPointLocalWindow)[],
```

Comment:
This is incorrect, we should type these as only `WebpackPluginEntryPointPreloadOnly[]`.  If typescript disagrees we should map `entry` into a `preloadOnlyEntry` manually on lines 197 and 206

Reply:
can you rephrase what you mean here